lib/postgrex.ex

```diff
@@ -62,6 +62,8 @@ defmodule Postgrex do
     * `:pool` - The pool module to use, see `DBConnection`, it must be
     included with all requests if not the default (default:
     `DBConnection.Connection`);
+    * `:null` - The atom to use as a stand in for postgres' `NULL' in encoding
```

Comment:
Should be markdown **`**NULL**`** not the erlang doc style **`**NULL**'**, same error in the other doc additions in this module.